util/getCurrentMonth.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { testDateAugust } from '../nrfcloud/calculateCosts.spec'
```

Comment:
Don't re-use simple constants (so the other test can change it without affecting this test).